README.md

```diff
@@ -1,9 +1,11 @@
-Luanti (formerly Minetest)
-==========================
-
-![Build Status](https://github.com/luanti-org/luanti/workflows/build/badge.svg)
-[![Translation status](https://hosted.weblate.org/widgets/minetest/-/svg-badge.svg)](https://hosted.weblate.org/engage/minetest/?utm_source=widget)
-[![License](https://img.shields.io/badge/license-LGPLv2.1%2B-blue.svg)](https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html)
+<div align="center">
+    <img src="textures/base/pack/logo.png">
```

Comment:
I suggest using `misc/luanti-xorg-icon-128.png` here as it isn't so huge.

Reply:
I dissagree